src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -346,4 +351,11 @@ codeunit 149005 "BCPT Line"
         Parm := format(FldRef.Value, 0, 9);
         exit(true);
     end;
+
+    local procedure RaiseOnBeforeBCPTLineAddLogEntry(var BCPTLine: Record "BCPT Line"; var Operation: Text; var ExecutionSuccess: Boolean; var Message: Text; var IsHandleld: Boolean)
```

Comment:
I still do no understand why you have this method. You don't need it.

Reply:
not that it matters, but there's also a typo in IsHandled

Reply:
Removed metod